tests/client/amo/components/TestAddonDetail.js

```diff
@@ -177,13 +177,14 @@ describe('AddonDetail', () => {
     const allowedTags = [...allowedDescriptionTags];
     // Ignore <br/> since it's checked elsewhere.
     allowedTags.splice(allowedTags.indexOf('br'), 1);
-
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
Thanks for this comment: https://github.com/mozilla/addons-frontend/pull/1379#issuecomment-261549708

I was just about to say "there are an awful lot of these exceptions" but now I see why.

That said, could you add a link to your comment next to them to explain why? Or file a code-quality issue to remove them. I agree with the rationale, just don't want it to get lost.